### PR TITLE
Add new BIOS version '12-07' for 'snes4sl' [Victor Fernandez (City Game), ClawGrip]

### PR DESCRIPTION
**Pull request by clawgrip**

### CODE REVIEW

**Comment by startaq** [comment]
Please use MAME's `ROM_SYSTEM_BIOS` support instead of adding a new set.

**Comment by clawgrip** [comment]
You're right!

Will do.

Thanks. Best regards: ClawGrip
________________________________
De: Dirk Best <notifications@github.com>
Enviado: jueves, 28 de enero de 2021 10:46
Para: mamedev/mame <mame@noreply.github.com>
Cc: ClawGrip <clawgrip@hotmail.com>; Author <author@noreply.github.com>
Asunto: Re: [mamedev/mame] New clones marked as NOT_WORKING (#7714)


Please use MAME's ROM_SYSTEM_BIOS support instead of adding a new set.

File: src/mame/drivers/snesb51.cpp [hunk 1]
```diff
@@ -251,7 +254,7 @@ ROM_START( snes4sln )
...
-GAME( 199?, mk3snes,  0,      mk3snes,  mk3snes, snesb51_state, empty_init, ROT0, "bootleg", "Mortal Kombat 3 (SNES bootleg)",        MACHINE_IS_SKELETON )
-GAME( 1993, snes4sl,  0,      snes4sl,  snes4sl, snesb51_state, empty_init, ROT0, "bootleg", "SNES 4 Slot arcade switcher",           MACHINE_IS_SKELETON )
-GAME( 1994, snes4sln, 0,      snes4sln, snes4sl, snesb51_state, empty_init, ROT0, "bootleg", "SNES 4 Slot arcade switcher (NBA Jam)", MACHINE_IS_SKELETON )
+//    YEAR  NAME      PARENT MACHINE   INPUT    CLASS          INIT        ROT   COMPANY    FULLNAME                                 FLAGS
+GAME( 199?, mk3snes,  0,     mk3snes,  mk3snes, snesb51_state, empty_init, ROT0, "bootleg", "Mortal Kombat 3 (SNES bootleg)",        MACHINE_IS_SKELETON )
+GAME( 1993, snes4sl,  0,     snes4sl,  snes4sl, snesb51_state, empty_init, ROT0, "bootleg", "SNES 4 Slot arcade switcher",           MACHINE_IS_SKELETON )
+GAME( 1994, snes4sln, 0,     snes4sln, snes4sl, snesb51_state, empty_init, ROT0, "bootleg", "SNES 4 Slot arcade switcher (NBA Jam)", MACHINE_IS_SKELETON )
```
Comment (cuavas) [inline src/mame/drivers/snesb51.cpp:260]: This is unnecessary churn – the additional space was there so there’s a double space between “PARENT” and “MACHINE” so it’s clearer they’re separate.

Comment (clawgrip) [reply src/mame/drivers/snesb51.cpp]: Fixed, thx!

**Pull request merged**